CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 - Project prepared to be installed with `pip install`, so it can be reused in 
   the repository https://github.com/canonical/gatekeeper-repo-test
 
+## [v0.9.1] - 2024-06-28
+
+### Fixed
```

Comment:
Until released it should be noted under `Unreleased` (i.e., without a date heading) please. Once we make a release with this, it'll get a date and version